docs/source/reference/response.rst

```diff
@@ -0,0 +1,11 @@
+.. _ref-response:
+
+==================
+Response Reference
+==================
+
+botocore.response
+-----------------
+
+.. automodule:: botocore.response
```

Comment:
Won't this document any additional functions we add to this module?  I'd prefer we just call out StreamingBody for now.